data-raw/process_raw_data.R

```diff
@@ -0,0 +1,3 @@
+ebola <- readxl::read_xlsx("data-raw/pnas.1518587113.sd02.xlsx")
```

Comment:
These are now development dependencies and so potentially should go in the suggests section of the description